contrib/builder/rpm/amd64/generate.sh

```diff
@@ -111,15 +111,15 @@ for version in "${versions[@]}"; do
 	esac
 
 	case "$from" in
-		oraclelinux:6)
+		oraclelinux:6|amazonlinux:latest)
 			# doesn't use systemd, doesn't have a devel package for it
 			packages=( "${packages[@]/systemd-devel}" )
 			;;
 	esac
 
 	# opensuse & oraclelinx:6 do not have the right libseccomp libs
 	case "$from" in
-		opensuse:*|oraclelinux:6)
+		opensuse:*|oraclelinux:6|amazonlinux:latest)
```

Comment:
Not really, no. You could remove it so it was ready to build once you ship the library I guess.